Doc/reference/compound_stmts.rst

```diff
@@ -491,6 +492,9 @@ is semantically equivalent to::
 .. versionchanged:: 3.1
    Support for multiple context expressions.
 
+.. versionchanged:: 3.10
+   Support for using grouping parentheses to break the statement in multiple lines.
```

Comment:
I'm not sure this is clear out of context. Maybe it's worth changing in the example `with A() as a, B() as b:`  to multiple lines to make it clearer?

Reply:
I added a new small sentence with the multi-line example so this is more contextualized